src/elements/content-sidebar/additional-tabs/AdditionalTab.js

```diff
@@ -94,9 +104,14 @@ class AdditionalTab extends React.PureComponent<Props, State> {
         });
 
         const tooltipText = isDisabled ? this.getDisabledReason() : title;
+        const isFtuxTooltipVisible = !isLoading && !!ftuxTooltipData;
 
         return (
-            <Tooltip position="middle-left" text={tooltipText}>
+            <AdditionalTabFtuxTooltip
+                defaultTooltipText={tooltipText}
+                ftuxTooltipData={ftuxTooltipData}
+                isFtuxVisible={isFtuxTooltipVisible}
```

Comment:
I think we can do `isFtuxVisible={!isLoading}` and check for the existence of `ftuxTooltipData` inside AdditionalTabFtuxTooltip.